modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -707,3 +703,12 @@ sda.enkf.multisite <- function(settings,
   } ### end loop over time
   
 } # sda.enkf
+
+#' @title loadRData
+#' @param fileName path to the Rdata.
+#' Enable the feature of passing loaded object into new variable.
+loadRData <- function(fileName){
+  #loads an RData file, and returns it
+  load(fileName)
+  mget(ls()[ls() != "fileName"])
+}
```

Comment:
This looks like a reimplementation of `PEcAn.utils::load_local` -- I recommend using that instead.

Reply:
I reverted back because this is much more complicated than I thought.